packages/store/tests/select.spec.ts

```diff
@@ -113,7 +113,7 @@ describe('Select', () => {
       comp.componentInstance.state.subscribe();
     } catch (e) {
       expect(e.message).toEqual(
-        `Cannot assign to read only property '__state__selector' of object '[object Object]'`
+        `Cannot add property __NG_ELEMENT_ID__, object is not extensible`
```

Comment:
:thinking:

Reply:
Yes) Now Angular now checks the freezing of component classes, can we remove this test?

Reply:
This test is now exhibiting different behavior because of an angular change.
Are you able to just the test so that it tests what the original test was testing?

Reply:
`Are you able to just the test so that it tests what the original test was testing?`
No